src/components/packages/ChallengeRow.jsx

```diff
@@ -20,7 +19,11 @@ import * as constants from '~/constants'
 export const challengeRowQuery = gql`
   query challengeRowQuery($challengeId: String!, $uri: String!) {
     metadata(uri: $uri) @client {
-      ...md
+      id
+      title
+      body
+      html_url
```

Comment:
It's too bad we can't rename these vars coming from APIs to be consistent (ie. htmlUrl)

Reply:
Ah wait....of course we can.  It's GraphQL